Sources/AssetGen/Infrastructure/SwiftLocalizeGenerator/Processor/PlaceholderType.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+enum PlaceholderType {
+  case string
+  case int
+  case double
+  case any
+  
+  init(for placeholder: String) {
```

Comment:
Well, I see that it is better to update the `Placeholder` struct to have the type of `PlaceholderType` instead of a String, since the PlaceholderType  should be created from regex match string

```
  struct Placeholder {
    let index: Int
    let placeholderType: PlaceholderType
  }
```

Reply:
Consider renaming `PlaceholderType` to `Placeholder.Variant` That way you can use it in the placeholder directly since `type` is reserved word.
```swift
struct Placeholder {
  let index: Int
  let variant: Variant
}

extension Placeholder {
  enum Variant { ... }
}
```